### PR TITLE
feat: updating workflows and tasks to latest varsion

### PR DESCRIPTION
**Pull request by Michael-Kruggel**

### CODE REVIEW

**Review by corang — APPROVED** [review]

**Review by mkborland — APPROVED** [review]

File: .github/workflows/pull-request-conditionals.yaml
Comment (robmcelvenny) [inline .github/workflows/pull-request-conditionals.yaml:1]: file not needed

**Review by robmcelvenny — CHANGES_REQUESTED** [review]
once change is made, LGTM

**Comment by Michael-Kruggel** [comment]
Deleted the file @robmcelvenny

**Review by robmcelvenny — DISMISSED** [review]
LGTM

**Review by corang — APPROVED** [review]

**Pull request merged**